integrations/visual-website-optimizer/HISTORY.md

```diff
@@ -1,3 +1,14 @@
+2.5.0 / 2019-10-03
```

Comment:
Let's do only one version bump (like #12) and release both changes at the same time.